bpfman/src/command.rs

```diff
@@ -1520,5 +1498,5 @@ impl Program {
 // it.
 #[derive(Debug, Clone)]
 pub(crate) struct BpfMap {
-    pub(crate) used_by: Vec<u32>,
+    pub(crate) _used_by: Vec<u32>,
```

Comment:
If this is the only field in the struct, and it's unused... does that mean the whole struct can be removed?

Reply:
Good catch meant to remove this 👍